apps/showcase/src/main/java/org/apache/struts2/showcase/fileupload/MultipleFileUploadUsingArrayAction.java

```diff
@@ -32,9 +32,9 @@
  */
 public class MultipleFileUploadUsingArrayAction extends ActionSupport {
 
-	private File[] uploads;
-	private String[] uploadFileNames;
-	private String[] uploadContentTypes;
+	private File[] uploads = new File[0];
```

Comment:
This looks odd. Why new?

Reply:
Hello @aleksandr-m .
The new initializers were the quickest fix (and equivalent to what is done in  `MultipleFileUploadUsingListAction.java`) for a boundary-condition bug in processing the action. 

Without allocated fields (via new in initializers), the following steps resulted in an HTTP 500 (due to NPE in `upload()`):
1) Choose Multiple File Upload (Array) from File menu.
2) Click on Submit (without adding any files to the upload).

With this fix both types of multiple file upload attempts can process empty payloads without failing.  The upload() methods of both classes could be changed as an alternative, but this was the least-invasive change.